### PR TITLE
function for dropdown menu and some minor changes added

### PR DESCRIPTION
**Pull request by magu1216**
I've added the burger menu, morr styling coming soon.
Base file structure completed.

### CODE REVIEW

File: src/components/BurgerButton.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React, { useState } from "react";
+import MenuIcon from "@material-ui/icons/Menu";
+import CloseIcon from "@material-ui/icons/Close";
+
+export default function BurgerButton(props) {
+  return (
+    <div>
```
Comment (miriamhaenle) [inline src/components/BurgerButton.js:7]: Consider using fragments `<> </>` here instead of divs

File: src/components/Navigation.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react";
+//import { Link } from "react-router-dom";
```
Comment (miriamhaenle) [inline src/components/Navigation.js:2]: Be carefull, it is not considered good coding style to push commented out stuff on main / master :)

File: src/components/Navigation.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+import styled from "styled-components";
+import BurgerButton from "./BurgerButton";
+
+export default function Navigation() {
+  const [dropdown, setDropdown] = useState(false);
+
+  //toogle dropdown
+  const showDropdown = () => setDropdown(!dropdown);
+  console.log(dropdown);
```
Comment (miriamhaenle) [inline src/components/Navigation.js:11]: no console log on main

File: src/components/Header.js [hunk 4]
```diff
@@ -1,14 +1,19 @@
...
+import Navigation from "./Navigation";
 
 export default function Header() {
   return (
     <StyledHeader>
       <h2>weSPAWN</h2>
       <h1>Profile</h1>
-      <div>Placeholder</div>
-      <Burger />
+      <Router>
```
Comment (miriamhaenle) [inline src/components/Header.js:11]: Higher Order Components like Router should usually be put into a higher component and wrap around the components. Consider putting it into app.js

File: src/components/Header.js [hunk 5]
```diff
@@ -1,14 +1,19 @@
 import React from "react";
 import styled from "styled-components";
-import Burger from "./Burger";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
+import Navigation from "./Navigation";
 
 export default function Header() {
   return (
     <StyledHeader>
```
Comment (miriamhaenle) [inline src/components/Header.js:8]: Consider renaming the whole component to `AppHeader` and then you can just use `<Header> ` for your styled component.

**Review by miriamhaenle — COMMENTED** [review]
Great job! Keep going :)

**Review by FabianAlleblas — COMMENTED** [review]
Nice one, but I don't get your usecase of the Router in the header component, maybe I just dont see it 😅 . To keep your code clean avoid comments and unused code.

File: src/components/Header.js (hunk 4 above)
Comment (magu1216) [reply src/components/Header.js:11]: good point, thank you!

File: src/components/Header.js [hunk 6]
```diff
@@ -1,14 +1,19 @@
...
       <h2>weSPAWN</h2>
       <h1>Profile</h1>
-      <div>Placeholder</div>
-      <Burger />
+      <Router>
+        <Navigation />
+        <Switch>
+          <Route path="/" />
+        </Switch>
+      </Router>
```
[the remark this answers is not in the document]
Comment (magu1216) [reply src/components/Header.js:16]: Was playing around with Router. That was the rest of it :d

**Pull request merged**